content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -224,6 +226,8 @@ Examples:
 
 **Note**: The Serilog library requires message property names to be valid C# identifiers. The required property names are: `dd_env`, `dd_service`, `dd_version`, `dd_trace_id`, and `dd_span_id`.
 
+**Note**: If you are using Serilog through ILogger, see the [ILogger section](?tab=microsoftextensionslogging#configure-log-collection) to configure these properties via `BeginScope()`.
```

Comment:
This is the ILogger section: https://docs.datadoghq.com/tracing/connect_logs_and_traces/dotnet/?tab=microsoftextensionslogging#configure-log-collection

Reply:
Two notes:

1. I recommend moving this note out of the Serilog example and into the Manual injection section, because this applies to multiple logging libraries, not just Serilog.
2. Since this will be directly above the examples tabs, I think you can remove the link entirely.

Reply:
Done